### ISSUE TITLE
Investigate and resolve accessibility issues raised on sortable table component and search component

### ISSUE THREAD

**Issue by robertjmccarthy** [ministryofjustice/moj-frontend#1110]
## What
Document the accessibility issue for the components on the site, resolve the identified issue, implement the update, and notify users accordingly.

## Why
Slack message on support channel flagged an accessibility issue raised: https://mojdt.slack.com/archives/CH5RUSB27/p1737129000538939

## Who needs to be involved
- Accessibility Specialist (leading)
- Developer (supporting)

## Done
When a resolution is found, applied and communicated out to users (and specifically communicated to the team that raised the issue)

**Comment by helennickols** [comment]
@Whersi Hi, could you add the accessibility info from Ben here please so that I can view it? Thanks

**Comment by helennickols** [comment]
@chrispymm Do you know the fix date? It will go into this template:
![Image](https://github.com/user-attachments/assets/1fab1071-ea26-4e56-8747-1ec9852be55b)

@robertjmccarthy @jbarnardmoj Is the [notification badge](https://design-patterns.service.justice.gov.uk/components/notification-badge/) built into the template yet? I was proposing we could publish the accessibility statement first and then add the badge. WDYT? Or easier to do it all together? The content is in progress...

**Comment by robertjmccarthy** [comment]
Accessibility issue information added to component pages for sortable table and search.

### To be done
- [ ] Fix found and applied to both components
- [ ] Information to be removed from components once fix applied

**Comment by chrispymm** [comment]
### Update
**Sortable Table**
* Sortable table fix is relatively easy, but is likely a breaking change to the component.
* Given our new component statuses/roadmap we need a team discussion around the priority/timeline for this fix.  We may choose to leave it until we officially review and update the component.
* If we choose to. delay, we will then need to update the guidance site to reflect the new timeline.

**Search**
* The close button in question is actually added by the browser as it is an `input type="search"` 
* @benproctor-rogers to contact uservision to see if this is still a fail (probably)
* If it is, the fix is a simple one - to switch to an `input type="text"`

**Comment by benproctor-rogers** [comment]
The sortable table issue is now fixed.

I met with User Vision (MoJ third party audit contractor)on 10/04/25 and discussed the search box button issue. User Vision are of the mind that this is an accessibility issue and will continue to flag it if there is not a fix. They suggest disabling the browser added delete text button and replacing with a custom button to delete the text from the search box.

**Comment by Whersi** [comment]
Closing this ticket as the sortable table component has been fixed and the search component has it's own ticket now https://github.com/ministryofjustice/moj-frontend/issues/1382
